continuous-delivery/build-wheels-musllinux-1-1-x86_64.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#assumes image based on musllinux_1_1 + extras (pip)
+set -ex
+
+/opt/python/cp39-cp39/bin/python ./continuous-delivery/update-version.py
+
+/opt/python/cp37-cp37m/bin/python setup.py sdist bdist_wheel
+auditwheel repair --plat musllinux_1_1_x86_64 dist/awscrt-*cp37*.whl
+
+/opt/python/cp38-cp38/bin/python setup.py sdist bdist_wheel
+auditwheel repair --plat musllinux_1_1_x86_64 dist/awscrt-*cp38*.whl
+
+/opt/python/cp39-cp39/bin/python setup.py sdist bdist_wheel
+auditwheel repair --plat musllinux_1_1_x86_64 dist/awscrt-*cp39*.whl
+
+/opt/python/cp310-cp310/bin/python setup.py sdist bdist_wheel
+auditwheel repair --plat musllinux_1_1_x86_64 dist/awscrt-*cp310*.whl
+
+/opt/python/cp311-cp311/bin/python setup.py sdist bdist_wheel
+auditwheel repair --plat musllinux_1_1_x86_64 dist/awscrt-*cp311*.whl
+
```

Comment:
trivial: Can you add the following comment to all these copy/pasty files? Just to help out any future maintainers

```suggestion

# Don't need to build wheels for Python 3.12 and later.
# The 3.11 wheel uses the stable ABI, so it works with newer versions too.

```

I guess it's weird, since we haven't merge https://github.com/awslabs/aws-crt-python/pull/479 yet ... maybe add the comment to the existing files in that PR, and to the new files in this PR? or just add it to all of them here 🤷‍♀️